README.md

```diff
@@ -1 +1,34 @@
-# School-Library-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# MorseCode-decoder
+
+> Imagine that you are the librarian of OOP University, and you need a tool to record what books are in the library and who borrows them. And all of this have been built in a beautiful and well-organized way!
+
+
+## Built With
+
+- `Ruby`
+
+## Author
+
+👤 **Sanja Mandic**
+
+I am an aspiring web developer from Serbia.
+- GitHub: [@sanjaGit](https://github.com/Sanja969)
+- Twitter: [@sanjaTwit](https://twitter.com/SanjaMandic42)
+- LinkedIn: [@sanjaIn](https://linkedin.com/in/sanja-mandic-823995a2/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Sanja969/Decode-a-Morse-code-message/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
You did a great job adding the README file, however, the title is wrong, the description is missing and also some other sections like setup and prerequisites are missing. Kindly add these sections. Remember it is super important to have a good Readme file.

Reply:
@muneebulrehman 

- Thank you for noticing that I made mistake in the readme file, I didn't change the title when I used the readme from my last project. I added instructions for installing.

- I think in this case would be weird if I committed more, I just made a few lines of code. Committing more often is better, but in this case really?